Makefile

```diff
@@ -106,6 +106,24 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
+# find or intall autorest
+find-autorest:
+ifeq (, $(shell which autorest))
+	@{ \
+	set -e ;\
+	NODE_PATH=$(shell ./find-node-or-install) \
+    PATH=$$NODE_PATH:$(shell echo $$PATH) \
```

Comment:
I think something's wrong with indentation. Are these tabs?

Reply:
Right, that line was erroneously using spaces. I have fixed it.